edweiss-firebase/functions/src/functions/fcm/sendFCMPage.ts

```diff
@@ -0,0 +1,28 @@
+import sendFCMMessage from 'actions/sendFCMMessage';
+import { FCMCommunication } from 'model/users';
+import { onAuthentifiedCall } from 'utils/firebase';
+import { Collections, getDocument } from 'utils/firestore';
+import { fail, ok } from 'utils/status';
+
+export const sendFCMPage = onAuthentifiedCall(FCMCommunication.Functions.sendFCMPage, async (userId, args) => {
+
+
```

Comment:
You can add more test of args, by exemple the number of page : 

```
if (args.page < 0)
    return fail("invalid_page");
```

Reply:
thank you I will add it